### DISCUSSION TITLE
Make LED strip controller bluetooth only?

### DISCUSSION THREAD

**Discussion by ballo** [arendst/Tasmota#21425] [General]
Hello. 
I fear I made a mistake buy a "bluetooth" LED strip controller model "WB5". Apparently this is some God-awful "tuya" cloud-only device. Is there any way to make this a bluetooth only device or did I just screw up? Thanks.

**Comment by sfromis** [comment]
If what you mean is https://templates.blakadder.com/WB5.html that device is not relevant to Tasmota unless you replace the CB3S module inside of it. What you can do with it in "Tuya hell" is not something I've concerned myself with finding out, but I think that there's other open source firmware that you can replace with, without brain surgery.

**Comment by HofeBY** [comment]
@ballo take a look at
https://github.com/openshwprojects/OpenBK7231T_App?tab=readme-ov-file

**Comment by ballo** [comment]
It isn't clear that even if I do this (I have a reflow station and only need the programmer) if this can be used as a bluetooth only device.

**Reply by sfromis** [reply]
What could make it Bluetooth-only would not be swapping the Wifi module, but depends on what firmware you run. And Tasmota would not make it Bluetooth-only.
